content/developer/tutorials/getting_started/07_basicviews.rst

```diff
@@ -218,7 +218,18 @@ services *OR* have a unit price which is *NOT* between 1000 and 2000'::
             ('unit_price', '>=', 1000),
             ('unit_price', '<', 2000)]
 
-.. exercise:: Add filter and Group By.
+.. note:: XML does not allow ``<`` and ``&`` to be used inside XML
+    elements. To avoid parsing errors, entity references should be used:
+    ``&lt;`` for ``<`` and ``&amp;`` for ``&``. Other entity references
+    (``&gt;``, ``&apos;`` & ``&quot;``) are optional.
+
+    Example:
```

Comment:
This could use the [`example`](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#example) directive.

Reply:
Didn't know about that one yet, added it.